wallet-core/bulding.md

```diff
@@ -0,0 +1,86 @@
+# Building
+
+Find here intructions for building the Wallet Core library locally.
+
+## Platforms
+
+The following *target platforms* are supported:
+
+* **iOS** -- Swift language is used by the library.
+* **Android** -- Kotlin language is used by the library.
```

Comment:
I think we use `Java` is used, not Kotlin

Reply:
I think we use both Java and Kotlin, for unit tests Kotlin.